main.js

```diff
@@ -0,0 +1,107 @@
+// let myJSON = {
```

Comment:
To read the Json data, I imported it into my JS file. 

import NAME from "./file path to json"
ie. mine was import resume from "./data/resume.json";

then later on to iterate through the .json you just refer to the variable. resume.work, resume.basics etc.